tests/seed_isort_config_test.py

```diff
@@ -295,7 +295,7 @@ def test_exclude(tmpdir):
 def test_returns_zero_no_changes(tmpdir):
     with tmpdir.as_cwd():
         cfg = tmpdir.join('.isort.cfg')
-        cfg.write('[settings]\nknown_third_party=cfgv\n')
+        cfg.write_binary(b'[settings]\nknown_third_party=cfgv\n')
```

Comment:
don't change these please -- this is unrelated to your patch

Reply:
With them being as they were they would fail when run on Windows

Reply:
As now we preserve line endings and `cfg.write` is using system line ends.

Reply:
but the same is true of `.read()`, no?

Reply:
yes, it is -- I just tested this on windows:

```python
def test(tmpdir):
    f = tmpdir.join('f')
    f.write('foo\nbar\n')
    assert f.read() == 'foo\nbar\n'
    assert f.read_binary() == b'foo\r\nbar\r\n'
```

Reply:
Yes. But because of https://github.com/asottile/seed-isort-config/pull/56/files#diff-1ee916b8a222e880354d894c92719eefR125 read now returns lines with `\r\n` on windows. So `assert` is no longer valid

Reply:
You are right. Problem is different. Give me a moment

Reply:
if the test is failing with `write(...)` then the change is currently broken and needs fixing

Reply:
Yes. It's breaks things on 
```            third_party_s = dump(sorted(third_party))
            replacement = fr'known_third_party\1=\2{third_party_s}'
            new_contents = THIRD_PARTY_RE.sub(replacement, contents)```

Reply:
I don't think that's quite the right explanation, I think it's because the regex is capturing `\r`

Reply:
That what I meant